File: Part2/Python/BasicConcepts/BasicConcepts_Uninitialized.py
def main():
    # I think in some programs this will cause a crash or failure to compile, so it will be in a separate program.
    # Python doesn't have uninitialized variables - this will cause NameError
    print ( "myCharacter:", myCharacter, "as int:", ord ( myCharacter ) )

if __name__ == "__main__":
    main()

File: Part2/Rust/BasicConcepts/BasicConcepts_Uninitialized.rs
fn main()
{
    // I think in some programs this will cause a crash or failure to compile, so it will be in a separate program.
    // Rust doesn't allow uninitialized variables - this will cause compile error
    let my_character: char;
    println!("myCharacter: {} as int: {}", my_character, my_character as u32);
}
